src/explorer/AzureAccountTreeItem.ts

```diff
@@ -14,4 +17,38 @@ export class AzureAccountTreeItem extends AzureAccountTreeItemBase {
     public createSubscriptionTreeItem(root: ISubscriptionContext): SubscriptionTreeItem {
         return new SubscriptionTreeItem(this, root);
     }
+
+    public async loadMoreChildrenImpl(clearCache: boolean, context: IActionContext): Promise<AzExtTreeItem[]> {
+        const children: AzExtTreeItem[] = await super.loadMoreChildrenImpl(clearCache, context);
+
+        const loginSession: string | undefined = ext.context.globalState.get(TrialAppLoginSession);
+
+        if (loginSession) {
+            const ti: AzExtTreeItem[] = await this.createTreeItemsWithErrorHandling(
+                [loginSession],
+                'trialAppInvalid',
+                async (source: string): Promise<AzExtTreeItem> => {
+                    return await TrialAppTreeItem.createTrialAppTreeItem(this, source);
+                },
+                (_source: unknown): string => {
+                    return 'Could not import trial app';
```

Comment:
Is there not an error message you could return here instead?

Reply:
Looking at other examples of `createTreeItemsWithErrorHandling` it doesn't look like we are supposed to return an error message there. Instead the error is put as a child of the node. 

For example: 

https://github.com/microsoft/vscode-azureappservice/blob/83c6f0df8ad80e6d1775fade38070b32b3b32fc6/src/explorer/SubscriptionTreeItem.ts#L60